app/models/auction_ended.rb

```diff
@@ -0,0 +1,14 @@
+class AuctionEnded
```

Comment:
seems like this should be a service obj rather than regular model? (given `perform` method)